packages/firestore/src/protos/update.sh

```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o nounset
```

Comment:
Unofficial strict mode for bash is

```
set -euo pipefail
IFS=$'\n\t'
```

Rationale here: http://redsymbol.net/articles/unofficial-bash-strict-mode/

It's worth just putting this at the top of every script.

Reply:
done